assets/css/base.css

```
@@ -553,6 +553,58 @@ footer {
   text-shadow: #ffffff 0 0 20px !important;
 }
 
+/* ポップアップ全体のスタイル */
+#terms-popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.7);
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-clip



(lint/suspicious/noShorthandPropertyOverrides)